src/components/hoc/homeLayout/index.css

```diff
@@ -1,19 +1,19 @@
 .header {
-    min-height: 10vh;
-    background-color: #fff;
+    min-height: 10vh !important;
+    background-color: #fff !important;
 }
 
 .content {
-    min-height: 80vh;
-    text-align: center;
-    background-color: #fff;
+    min-height: 80vh !important;
+    text-align: center !important;
+    background-color: #fff !important;
 }
 
 .footer {
-    min-height: 10vh;
-    background-color: #fff;
+    min-height: 10vh !important;
+    background-color: #fff !important;
 }
 
 .center_form {
-    margin: auto;
+    margin: auto !important;
```

Comment:
Is there a way to target these without having to use !important?

Reply:
Very important that you do not use `!important`. Use css specificity to make sure that you can target these various elements without `!important`